src/main/webapp/election-info.js

```diff
@@ -256,4 +256,9 @@ function addPollingStationMarker(map, position, title, description) {
   marker.addListener('click', () => {
     infoWindow.open(map, marker);
   });
-}
\ No newline at end of file
+}
+
+Handlebars.registerHelper('stripUrl', function(urlString){
```

Comment:
Can you add some comments / documentation on what this is supposed to be used for?

I'm guessing it's meant to strip off the "http://" part of the url

Reply:
done